.travis.yml

```diff
@@ -29,6 +29,7 @@ before_install:
 
 install:
   # Add Godeps dependencies to GOPATH and PATH
+   - eval "$(curl -sL https://raw.githubusercontent.com/travis-ci/gimme/master/gimme | GIMME_GO_VERSION=1.4 bash)"
```

Comment:
Hello!  I'm working on a solution to have a go binary in `$PATH` for all values of `language:`, but wanted to let you know that this line *should* be replaceable with:

``` yaml
  - eval "$(gimme 1.4)"
```